pkg/agent/plugin/keymanager/disk/disk_test.go

```diff
@@ -67,6 +67,8 @@ func TestDisk_Configure(t *testing.T) {
 	_, e := plugin.Configure(ctx, cReq)
 	assert.NoError(t, e)
 	assert.Equal(t, "foo/bar", plugin.dir)
+	assert.DirExists(t, "foo/bar")
+	os.RemoveAll("foo")
```

Comment:
we should use a temp directory instead of littering the source with `foo/bar`

Reply:
okay. I will fix it to use a temp directory.

Reply:
I fixed it with https://github.com/spiffe/spire/pull/899/commits/60b8ea5dd34a8d9d3784fbbda6645613aa272e44 .